W-14696736-Update build-environment.adoc (#67)
mulesoft/docs-accelerators: accelerators-home/modules/ROOT/pages/build-environment.adoc

* Latest Maven 3.8.x release (for command-line deployments)
* Postman 9.x or higher (for API testing)

[NOTE]
====

> If you use Anypoint Studio 7.16 and later, you will need to override the Maven settings to use Maven version 3.8.x. To override the settings:

Reviewer: ```suggestion
If you use Anypoint Studio 7.16 and later, you will need to override the Maven settings to use Maven version 3.8.x. To override the settings:

```

* Latest Maven 3.8.x release (for command-line deployments)
* Postman 9.x or higher (for API testing)

[NOTE]
====

> If you use Anypoint Studio 7.16 and later, you will need to override the Maven settings to use Maven version 3.8.x. To override the settings:

Reviewer 2: ```suggestion
If you use Anypoint Studio 7.16 and later, you must override the Maven settings to use Maven version 3.8.x. To override the settings:
```

Reviewer 2: To avoid will/future tense
